scripts/_common.sh

```diff
@@ -14,14 +14,15 @@ setup_incus() {
         ynh_add_config --template="incus-preseed-cluster.yml" --destination="/tmp/incus-preseed-cluster.yml"
         incus admin init --preseed < "/tmp/incus-preseed-cluster.yml"
         rm "/tmp/incus-preseed-cluster.yml"
-
-        incus config set core.https_address "[::]"
     else
         incus admin init --auto # --storage-backend=dir
     fi
 
     # Set a DNS tld because dnsmasq doesn't seem to like it?
     incus network set incusbr0 dns.domain=incus
```

Comment:
FYI, after removing incus, `incusbr0` still exists, so when you reinstall incus and reinitialize it, it will use the interface `incusbr1` instead, and this line make the install crash with the error:

> Error: Only managed networks can be modified

We can either remove `incusbr0` (with `ip link delete incusbr0`) in remove script or detect the interface used with something like: `incus profile show default | grep network: | cut -d':' -f2 | tr -d ' '`

Reply:
I'd prefer actively removing incusbr0, as a real cleaning step.